about.html

```diff
@@ -72,13 +72,26 @@ <h2>2023-19-5(Fri) <br><span class="span">CEO, Jalloh ~</span> StartUp TidCoin</
       <div class="image">
         <img class="mobile" src="images/image-mockups.png" alt="image-mockups" width="100%">
       </div>
+
     </section>
     <h3 class="parag1">Creative Commons history of Easybank and TidCoin</h3>
     <div class="section-creative-logo parag1">
       <p>Creative Commons history of Easybank and TidCoin - 2020 during the COVID-19 pandemic. Easybank and TidCoin emerged as two innovative fintech companies during the unprecedented times of the COVID-19 pandemic in 2020. As physical distancing measures were implemented globally, more and more people began relying on digital banking solutions for managing their finances. Easybank and TidCoin quickly adapted to this shift in behavior and saw tremendous growth in their user base during this period.</p>
   </div>
   <div class="logo-history parag1">
     <img src="images/logo.svg" alt="logo">
+  </div>
+  <div class="grids-web">
+    <div class="webinar">
+      <div class="container">
+        <p>2023</p>
+      </div>
+    </div>
+    <div class="webinar-2">
+      <div class="container">
+        <p>2024</p>
+      </div>
+    </div>
   </div>
     <section class="foods" id="blog">
         <h3>Latest Articles</h3>
```

Comment:
- [ ] Regarding to the last articles section great job for adding the extra effore on it but please let's stick with the requirements and make our design identical to the given design so kindly remove this section since it is not exist in the given design 💯🙏🏻🤩✨


![image](https://github.com/aradradev/easyBankTC/assets/58553711/040f91a9-186d-425c-b2da-75477fded972)

Reply:
@Reem-lab, Sorry, But last week I got approved, for this section, and I repeated the block for the assessment, to be honest guys, you're killing my motivation 😭😭😭.

Reply:
- [ ] Hello @aradradev I am so sorry but I have to follow the guidelines and I should ask for it since the previous reviewer asked to fix the about page layout and I explained in many comments to be more clear!